climpred/graphics.py

```diff
@@ -230,3 +236,219 @@ def plot_bootstrapped_skill_over_leadyear(bootstrapped, plot_persistence=True, a
+    lead_freq = get_lead_cftime_shift_args(hind.lead.attrs['units'], 1)[1]
+    hist = he.get_uninitialized()
+    if isinstance(hist, xr.Dataset):
+        hist = hist[variable]
+    obs = he._datasets['observations']
+
+    cmap = mpl.cm.get_cmap(cmap, hind.lead.size)
+    if not ax:
+        _, ax = plt.subplots(figsize=(10, 4))
+    if isinstance(hist, xr.DataArray):
+        if 'member' in hist.dims and not show_members:
+            hist = hist.mean('member')
+            member_alpha = 1
+            lw = 2
+        else:
+            member_alpha = 0.4
+            lw = 1
+        hist.plot(
+            ax=ax,
+            lw=lw,
+            hue='member',
+            color='gray',
+            alpha=member_alpha,
+            label='uninitialized',
+            zorder=hind.lead.size + 1,
+        )
+
+    for i, lead in enumerate(hind.lead.values):
+        h = hind.sel(lead=lead).rename({'init': 'time'})
+        if not show_members and 'member' in h.dims:
+            h = h.mean('member')
+            lead_alpha = 1
+        else:
+            lead_alpha = 0.5
+        h['time'] = shift_cftime_index(h.time, 'time', int(lead), lead_freq)
+        h.plot(
+            ax=ax,
+            hue='member',
+            color=cmap(i),
+            label=f'initialized: lead={lead}',
+            alpha=lead_alpha,
+            zorder=hind.lead.size - i,
+        )
+
+    linestyles = ['-', ':', '-.', '--']
+    if len(obs) > len(linestyles):
+        raise ValueError(f'Please provide fewer than {len(linestyles)} observations.')
+    if len(obs) > 0:
+        for i, (obs_name, obs_item) in enumerate(obs.items()):
+            if isinstance(obs_item, xr.Dataset):
+                obs_item = obs_item[variable]
+            obs_item.plot(
+                ax=ax,
+                color='k',
+                lw=3,
+                ls=linestyles[i],
+                label=f'reference: {obs_name}',
+                zorder=hind.lead.size + 2,
+            )
+
+    # show only one item per label in legend
+    handles, labels = ax.get_legend_handles_labels()
+    by_label = OrderedDict(zip(labels, handles))
+    ax.legend(
+        by_label.values(), by_label.keys(), loc='center left', bbox_to_anchor=(1, 0.5)
+    )
+    ax.set_title('')
+    return ax
+
+
+def plot_ensemble_perfect_model(
+    pm, variable=None, ax=False, show_members=True, cmap='tab10'
+):
+    """Plot datasets from PerfectModelEnsemble.
+
+    Args:
+        pm (PerfectModelEnsemble): PerfectModelEnsemble.
+        variable (str or None): `variable` to plot. Defaults to the first in data_vars.
+        ax (plt.axes): Axis to use in plotting. By default, creates a new axis.
+        show_members (bool): whether to display all members individually.
+            Defaults to True.
+        cmap (str): Name of matplotlib-recognized colorbar.. Defaults to 'tab10'.
+
+    Returns:
+        ax: plt.axes
+
+    """
+
+    _check_only_climpred_dims(pm)
+    if not variable:
```

Comment:
I think any reference to `None` should be `if ax is None`. I think this works, but this is also the language for `False`...